xchange-core/src/main/java/org/knowm/xchange/dto/trade/UserTrade.java

```diff
@@ -56,9 +56,10 @@ public UserTrade(
       String orderId,
       BigDecimal feeAmount,
       Currency feeCurrency,
-      String orderUserReference) {
+      String orderUserReference,
+      boolean liquidation) {
 
-    super(type, originalAmount, instrument, price, timestamp, id, null, null);
+    super(type, originalAmount, instrument, price, timestamp, id, null, null, liquidation);
```

Comment:
isn't `liquidation` the type of information that would only make sense to show in a `UserTrade`?

Reply:
No, in some exchanges it is a public information for every trade.